app/src/main/res/values-ru/strings.xml

```diff
@@ -547,4 +547,12 @@
   <string name="dialog_box_text_nomination">Почему %1$s должно быть удалено?</string>
   <string name="review_is_uploaded_by">%1$s загружено с помощью: %2$s</string>
   <string name="default_description_language">Язык описаний по умолчанию</string>
+  <string name="delete_reason_spam_selfie">Селфи</string>
```

Comment:
Removing hard coded strings and inserting them into strings.xml is perfectly fine. But we don't edit language specific strings.xml files manually. We just edit the default one ( app/src/main/res/values/strings.xml) as you also did. For outher languages we have translator updates. Please revert changes in this file.

Reply:
Done